src/python.grammar

```diff
@@ -110,7 +110,7 @@ skip whitespace {
 
   CommaSep(expr) { expr ("," expr?)* }
 
-  Body { SimpleStatement | indentedBody }
+  Body { SimpleStatement | ~indentedSimple indentedSimple | indentedBody }
```

Comment:
This seems to work in practice, but I wonder if this is the best way to get grammar-mode to do what I want? 

Note that without this clause the parser tries to apply `indentedBody` and fails because "\n" is followed by `pass` (in the example from the commit message).